src/server.ts

```diff
@@ -27,6 +28,13 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(rateLimit(limiter));
 
+// upon a successful authentication, a token will be granted to the user.
+// this token will need to be passed as part of the body of all requests
+// made to endpoints which require login, and will be checked if it is in here
+//
+// TODO: expire tokens after some amount of time
+app.set("tokenStore", {});
```

Comment:
so all tokens from all users will be kept in this tokenStore in-memory?

Reply:
yes -we'll have to figure out a way to delete old tokens at some point though so that the memory used doesn't grow forever

Reply:
can we store the tokens in the database instead? I think that's the typical approach - that way people don't get signed out if we restart the server. 
Typically you store the token and the date it was created, which allows us to invalidate a token after some time (like a day or a week later for example)